pkgs/by-name/ho/homebox/package.nix

```diff
@@ -0,0 +1,98 @@
+{
+  stdenvNoCC,
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  pnpm,
+  nodejs,
+}: let
+  pname = "homebox";
+  version = "0.10.3";
+  src = fetchFromGitHub {
+    owner = "hay-kot";
```

Comment:
This should probably use [sysadminsmedia/homebox](https://github.com/sysadminsmedia/homebox) as the original [hay-kot/homebox](https://github.com/hay-kot/homebox) has been [archived](https://github.com/hay-kot/homebox/issues/919).

Reply:
I'm aware, I just think it's a bit early to say which fork will live on and be supported long term. So I suggest waiting a bit longer, and then switching once any fork crystallizes out.
So far I think the changes [sysadminsmedia/homebox](https://github.com/sysadminsmedia/homebox) made are not critical, so I don't think we're missing out on much so far.

Reply:
switched to sysadminsmedia. Seems they're here to stay.